MonstieWash/Assets/Scripts/Player/PlayerHand_AnimationController.cs

```diff
@@ -0,0 +1,86 @@
+using NUnit.Framework.Constraints;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerHand_AnimationController : MonoBehaviour
+{
+    private enum AnimState { Stationary, Idle, Grab, Pet };
+    private AnimState m_currentAnim = AnimState.Idle;
+
+    [Tooltip("PlayerHand stationary animation")][SerializeField] private AnimationClip anim_stationary;
+    [Tooltip("PlayerHand idle animation")][SerializeField] private AnimationClip anim_idle;
+    [Tooltip("PlayerHand grab animation")][SerializeField] private AnimationClip anim_grab;
+    [Tooltip("PlayerHand petting animation")][SerializeField] private AnimationClip anim_pet;
+    
+    private Dictionary<AnimState, AnimationClip> m_handAnims = new Dictionary<AnimState, AnimationClip>();
```

Comment:
this one can be simplified
```suggestion
    private Dictionary<AnimState, AnimationClip> m_handAnims = new();
```

Reply:
Got it boss